### PR TITLE
 sys-cluster/ganglia-web: version bump 3.7.3

### PR DESCRIPTION
**Pull request by doctaweeks**
Addresses bug # 592080 and bug # 559658

### CODE REVIEW

**Comment by doctaweeks** [comment]
@whissi and @jsbronder please review and let me know if changes are necessary. Thanks.

File: sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+WEBAPP_MANUAL_SLOT="yes"
+inherit webapp eutils
```
Comment (Zlogene) [inline sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild]: what is purpose of `eutils` here?

File: sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+DEPEND="net-misc/rsync"
+RDEPEND="
+	${DEPEND}
+	${WEBAPP_DEPEND}
+	>=sys-cluster/ganglia-3.3.7[-minimal]
+	dev-lang/php[gd,xml,ctype]
+	virtual/httpd-php
+	media-fonts/dejavu"
+
+src_configure() {
```
Comment (Zlogene) [inline sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild]: why?

File: sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	return 0
+}
+
+src_compile() {
```
Comment (Zlogene) [inline sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild]: why?(2). Just empty function, I do not see the point for this to be here as the previous one

File: sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	return 0
+}
+
+src_install() {
+	webapp_src_preinst
+	cd "${S}"
```
Comment (Zlogene) [inline sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild]: `|| die` here

File: sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+	emake \
+		GDESTDIR="${MY_HTDOCSDIR}" \
+		DESTDIR="${D}" \
+		APACHE_USER=nobody \
+		install || die
```
Comment (Zlogene) [inline sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild]: drop `|| die` here

File: sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+	webapp_configfile "${MY_HTDOCSDIR}"/conf_default.php
+	webapp_src_install
+
+	fowners -R nobody:nobody /var/lib/ganglia-web/dwoo
+	fperms -R 777 /var/lib/ganglia-web/dwoo
+
+	dodoc AUTHORS README TODO || die
```
Comment (Zlogene) [inline sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild]: use `einstalldocs` (without further `|| die`)

**Comment by jsbronder** [comment]
466c912ccf53 sys-cluster/ganglia-web: drop old vulnerable versions
144acff1e4c6 sys-cluster/ganglia-web: version bump 3.7.3
ac2c33813831 sys-cluster/ganglia-web: Add github remote-id
62bab00f3e72 sys-cluster/ganglia-web: add proxy maintainer and proxy-maint

**Pull request closed without merging**